### PR TITLE
aim game ranged training

### PR DESCRIPTION
**Pull request by bombasticSlacks**
#### Summary
Content "Train Ranged Skills"

#### Purpose of change
Adds practice recipes for starting skills

#### Describe the solution
Looked for training tools for firearms online
https://www.nrafamily.org/content/3-ways-to-perfect-your-marksmanship-without-ammunition/

Adds a basic training for marksmanship to 1 just dry firing a gun
Adds beginner (0-3) training for each of pistols, shotguns, rifles, and smgs by using a laser hooked to the trigger of the gun and a target that displays your groupings.
Distributed the Aim Game kit
Gave all guns qualities related to this to make it not AWFUL to get the JSON for this working.

#### Testing
works in game

#### Additional context
image
image
image

Updated with batteries
image

### CODE REVIEW

File: data/json/items/generic/toys_and_sports.json [hunk 1]
```diff
@@ -234,6 +234,22 @@
...
   },
+  {
+    "type": "GENERIC",
+    "id": "aim_game",
+    "//": "lazerlyte trainer target was used as a reference",
+    "symbol": "*",
+    "color": "white",
+    "name": { "str": "aim game target practice kit" },
+    "category": "other",
+    "description": "A collection of lasers, and targets designed to help you practice shooting.  Where you shoot is dispayed on the targets.",
```
Comment (TheShadowFerret) [inline data/json/items/generic/toys_and_sports.json:245]:
```suggestion
    "description": "A collection of lasers, and targets designed to help you practice shooting.  Where you shoot is displayed on the targets.",
```

**Comment by RelMayers** [comment]
Ammo cost? Cause I don't seem to find it.

**Comment by bombasticSlacks** [comment]
> Ammo cost? Cause I don't seem to find it.

There isn't one. 

Screenshot_20220723-082810

**Comment by TheShadowFerret** [comment]
>There isn't one.

>BRIGHTNESS: Training pistol features a 5mW peak, 630-670nm, Class 3A visible red laser that is in conformance with and complies with 21 CFR 1040.10 with a max power output of "5mW" for a "Class 3R Red Laser" - the brightest beam allowed by law

Are you sure about that 🤔

**Comment by bombasticSlacks** [comment]
we don't track batteries on rail lasers normally, but the targets would as well. I'll generalize it to a battery for the kit

**Pull request merged**